Sources/NiceComponents/Components/ResizableImage.swift

```diff
@@ -59,26 +47,82 @@ public struct ResizableImage: View {
 
     public var body: some View {
         if let url = url {
-            KFImage(url)
-                .renderingMode(tintColor == nil ? .original : .template)
-                .resizable()
-                .placeholder {
-                    LoadingView(loadingStyle ?? .large)
-                }
-                .foregroundColor(tintColor)
-                .scaledToFill()
-                .frame(width: width, height: height)
-                .clipped()
+            AsyncImage(url: url) { image in
+                image.resizable()
+                    .renderingMode(tintColor == nil ? .original : .template)
+                    .aspectRatio(contentMode: contentMode)
+                    .frame(maxWidth: width, maxHeight: height)
```

Comment:
I think this means that you wouldn't be able to scale up an image, right? Is that something we want to do?

Reply:
Looks like the resizable modifier will make it scale up even though the frame modifier is suggesting to keep its size. Will change this back to set width/height explicitly. Setting the content mode to aspectFit fixes the issue we were seeing here before.